ES/ESmy016.xml

```diff
@@ -73,17 +74,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <incipit xml:lang="gez">
                               <locus target="#33v"/>ሰላም፡ ለዝከረ፡ ስምኪ፡ ስመ፡ ነከራ፡ ወተዓፅቦ፨
                               ወለርእስኪ፡ ሰላም፡ ብርሃነ፡ መለኮት፡ ዘይገለብቦ። ሰንበተ፡ ክርስቲያን፡ ቅድስት፡ ዕረፍተ፡ ኀጥአን፡ እምተመንድቦ፨
-                              ዕቀበኒ፡ ለመልአከ፡ ኀጕል፡ ከመ፡ ኢይኩን፡ መርከቦ፨<foreign xml:lang="en">
-                                 <locus target="#33v"/>Salutation to the reminding of
+                              ዕቀበኒ፡ ለመልአከ፡ ኀጕል፡ ከመ፡ ኢይኩን፡ መርከቦ፨
+                              <foreign xml:lang="en">
+                                 Salutation to the reminding of
                                  your name, the name of wonder and astonishment. And salutation to your
                                  head, which the light of the divinity covers. St Sabbath of Christians,
                                  repose of the sinners from the torments. Protect me so that I would not
                                  become a ship of the angel of perdition!</foreign>
                            </incipit>
                            <explicit xml:lang="gez">
-                              <locus target="#44r"/><hi rend="rubric">ሰን፡ ክ፡ ቅ፡</hi> ሀብኒ፡ ፍትሐ፡ ሣህልኪ፡
```

Comment:
why locus removed? it can stay (it is clear that it is in the end but still this is not a mistake to repeat it). I previously only commented that you do not need the locus twice, before the Geez citation and before the translation, but once is OK

Reply:
ok, restituted.